ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -40,6 +59,38 @@ const AssetListItem = ({
     )
     : null
 
+  const sendTokenButton = useMemo(() => {
+    if (tokenAddress == null) {
+      return null
+    }
+    return (
+      <Button
+        type="link"
+        className="asset-list-item__send-token-button"
+        onClick={(e) => {
+          e.stopPropagation()
+          sendTokenEvent()
+          dispatch(updateSendToken({
+            address: tokenAddress,
+            decimals: tokenDecimals,
+            symbol: tokenSymbol,
+          }))
+          history.push(SEND_ROUTE)
+        }}
+      >
+        {t('send')} {tokenSymbol}
```

Comment:
We probably want to add an i18n handler that takes `tokenSymbol` as an argument for this message, in case the object-verb order is different in some language.

Reply:
aw man I already had that ready, too! Thanks @rekmarks

Reply:
done.